File: JobSearchingWebApp/Endpoints/Korisnik/GetAll/KorisnikGetAllResponse.cs
namespace JobSearchingWebApp.Endpoints.Korisnik.GetAll
{
    public class KorisnikGetAllResponse
    {
        public List<KorisnikGetAllResponseKorisnik> Korisnici { get; set; }
    }

    public class KorisnikGetAllResponseKorisnik
    {
        public string Uloga { get; set; }

        public bool IsObrisan { get; set; } = false;
    }
}

File: front-end/src/app/endpoints/korisnik-endpoint/get-all/korisnik-get-all-response.ts
export interface KorisnikGetAllResponse {
  korisnici: {
    $values: KorisnikGetAllResponseKorisnik [];
    $id: number;
  }
}

export interface KorisnikGetAllResponseKorisnik {
  isObrisan: boolean,
  uloga: string,
  username: string
}
